### DISCUSSION TITLE
Hello Can A make a fork and add some things to your scripts and possibly make that public.

### DISCUSSION THREAD

**Discussion by JeflolxSpursf** [houdini101/AHK_Minecraft_Tools#3] [Ideas and Suggestions]
Idk if I will get an answer but it seems semi active.

EDIT: missed the top box. My bad.

**Comment by houdini101** [comment]
> Idk if I will get an answer but it seems semi active.
> 
> EDIT: missed the top box. My bad.

Hi! Of course you can fork this project and even edit the script, so long as you follow the notes embedded in the script.
